account_invoice_margin_sale_delivered_sync/models/sale.py

```diff
@@ -7,7 +7,7 @@
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    def _prepare_invoice_line(self):
+    def _prepare_invoice_line(self, **optional_values):
         vals = super()._prepare_invoice_line()
```

Comment:
You are breaking the method.. you must pass optional_values

Reply:
Right!, fixed.